extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -1150,6 +1150,236 @@ describe('amp-a4a', () => {
     });
   });
 
+  describe('#getKeyInfoSets_', () => {
+    let fixture;
+    let win;
+    let a4aElement;
+    beforeEach(() => {
+      return createAdTestingIframePromise().then(f => {
+        fixture = f;
+        win = fixture.win;
+        a4aElement = createA4aElement(fixture.doc);
+        return fixture;
+      });
+    });
+
+    function verifyIsKeyInfo(keyInfo) {
+      expect(keyInfo).not.to.be.null;
+      expect(keyInfo).to.have.all.keys(
+          ['serviceName', 'hash', 'cryptoKey']);
+      expect(keyInfo.serviceName).to.be.a('string').and.not.to.equal('');
```

Comment:
I didn't realize you could and... is that common practice as opposed to a separate expect item?